mix.exs

```diff
@@ -67,6 +67,7 @@ defmodule NervesHubUserAPI.MixProject do
       {:tesla, "~> 1.2.1 or ~> 1.3"},
       {:hackney, "~> 1.9"},
       {:x509, "~> 0.3"},
+      {:nerves_hub_ca_store, "~> 1.0.0", optional: true},
```

Comment:
```suggestion
      {:nerves_hub_ca_store, "~> 1.0.0"},
```

I think we need to enforce this being included, otherwise our default for `:ca_store` would fail